StorageAdapter/ValuesTest.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Helpers;
+using Helpers.Http;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace StorageAdapter
+{
+    [Collection("Storage Adapter Value Tests")]
+    public class ValuesTest
+    {
+        private readonly IHttpClient httpClient;
+
+        public ValuesTest()
+        {
+            var config = new ConfigurationBuilder()
+                .AddJsonFile("settings.json")
+                .AddEnvironmentVariables()
+                .Build();
+
+            string host = config["HOST"];
+            this.httpClient = new HttpClient();
+        }
+
+        /// <summary>
+        /// Integration test verifying an item can be posted to the values controller and 
+        /// subsequently retrieved.
+        /// </summary>
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void TheServiceCanPutAndRetrieveValues()
```

Comment:
nit: I would remove `TheService` from the method names here and elsewhere.

Reply:
Removed these.